db/flush_job.cc

```diff
@@ -227,12 +227,14 @@ Status FlushJob::Run(LogsWithPrepTracker* prep_tracker,
     prev_cpu_write_nanos = IOSTATS(cpu_write_nanos);
     prev_cpu_read_nanos = IOSTATS(cpu_read_nanos);
   }
-  autovector<MemTable*> purged_mems = {};
 
   if (db_options_.experimental_allow_mempurge &&
       (cfd_->GetFlushReason() == FlushReason::kWriteBufferFull) &&
       (!mems_.empty())) {
-    Status mempurge_s = MemPurge(purged_mems);
+    Status mempurge_s = MemPurge();
+    if (!mempurge_s.ok()) {
+      ROCKS_LOG_INFO(db_options_.info_log, "Mempurge process unsuccessful.");
```

Comment:
Please include status details, e.g. from ToString(). I'm sure you can find an example to copy.

It seems like an Aborted status should just be INFO but any other status is maybe WARN.